doc/source/conf.py

```diff
@@ -193,11 +193,21 @@
     "github_url": "https://github.com/pyansys/pymapdl",
     "show_prev_next": False,
     "show_breadcrumbs": True,
+    "collapse_navigation": True,
+    "use_edit_page_button": True,
     "additional_breadcrumbs": [
         ("PyAnsys", "https://docs.pyansys.com/"),
     ],
+    "icon_links": [
+        {
+            "name": "Support",
+            "url": "https://github.com/pyansys/pymapdl/discussions",
+            "icon": "fa fa-comment fa-fw",
+        },
+    ],
 }
 
+
```

Comment:
```suggestion
html_context = {
    "github_user": "pyansys",
    "github_repo": "pymapdl",
    "github_version": "main",
}
```

I think you are missing this still @germa89: See https://github.com/pyvista/pyvista/blob/e95b3ef6eff811b8469d0a97c992914bd26a28f4/doc/conf.py#L423-L428

Reply:
what thee.... how we have been running now? xD